src/parlant/bin/server.py

```diff
@@ -369,6 +406,9 @@ async def load_app(params: CLIParams) -> AsyncIterator[ASGIApplication]:
             evaluator=container[BehavioralChangeEvaluator],
         )
 
+        if params.module:
```

Comment:
`if params.modules` ... `load_modules(...)`

plural

Reply:
But isn't there a bug here? Like even if the param isn't specified but the modules come from the config file, you should load them.